content/applications/websites/ecommerce/managing_products/products.rst

```diff
@@ -0,0 +1,197 @@
+   - :doc:`../../../sales/sales/products_prices/products/import`
+
+Publish
+-------
+
+Upon creation, products are defaulted as :guilabel:`Unpublished` in your eCommerce catalog.
+**Unpublished products** are only visible to users with access rights to your eCommerce, whereas
+:guilabel:`Published` products are visible to all users. When you wish to render a product visible
+to all users, go to :menuselection:`Site --> Homepage --> Shop`, select the product, and enable the
+:guilabel:`Published` status in the top-right corner.
+
+Product page design
+===================
+
+Once a product is created, you can access its **product's page** through :menuselection:`Site -->
+Homepage --> Shop`, selecting the product, and clicking :guilabel:`Edit`. Here, you can change the
+**layout** of the page, enable **features**, add **content**, etc. Note that **enabled features**
+apply to *all* product pages.
+
+Features
+--------
+
+.. _ecommerce-features:
+
+In the **website builder** window, click :guilabel:`Customize` to enable additional features such as
+:guilabel:`Rating`, :guilabel:`Share`, etc.
+
+- :guilabel:`Customers`: :guilabel:`Rating` adds a rating option on the page; :guilabel:`Share` adds
+  share buttons via social media networks and email;
+- :guilabel:`Select Quantity`: if enabled, allows to choose the quantity added to cart;
+- :guilabel:`Tax Indication`: notifies if the price is **VAT included** or **excluded**;
+- :guilabel:`Variants`: shows all possible variants of the product as a :guilabel:`Products List`;
+  :guilabel:`Options` as selectable options to compose the variant yourself;
+- :guilabel:`Cart`: :guilabel:`Buy Now` adds a checkout button taking the customer directly to the
+  checkout page; :guilabel:`Wishlist` allows to add the product to a wishlist;
+- :guilabel:`Specification`: allows you to select where the **specification** category is displayed.
+   This only applies to products with **variants**.
+
+.. note::
+   - To allow **wishlists**, the option must be enabled in :menuselection:`Configuration -->
+     Settings --> Shop - Products --> Wishlists`;
+   - To access the :guilabel:`Variants` options, the :guilabel:`Product Variants` option must first
+     be enabled under :menuselection:`Configuration --> Settings --> Shop - Products`. You can learn
+     more about variants :doc:`here <../../../sales/sales/products_prices/products/variants>`.
+
+Layout
+------
+
+Within the same :guilabel:`Customize` tab as the :ref:`features <ecommerce-features>`, the layout
+configuration can be changed according to your needs.
+
+- :guilabel:`Images Width`: changes the *width* of the image product displayed on the page;
+- :guilabel:`Layout`: the :guilabel:`Carousel` layout displays a large, main image with smaller ones
+  underneath; whereas the :guilabel:`Grid` displays four images in a square layout (see pictures
+  below);
+- :guilabel:`Image Zoom`: if the zoom occurs on :guilabel:`Pop-up on Click`, when hovering over the
```

Comment:
The structure you propose here is not super clear to me as the options mentioned here are not all part of the "customize" section. 
![image](https://user-images.githubusercontent.com/85342080/207019342-11dcc3bc-4962-41cc-89ef-00ca31dc5f23.png)
![image](https://user-images.githubusercontent.com/85342080/207020483-baa82ec0-6f9c-452c-b3d7-f000511c0d83.png)

Reply:
Not sure where I got those extra options, I tried again and couldn't replicate it